types/whois-json/index.d.ts

```diff
@@ -0,0 +1,111 @@
+// Type definitions for whois-json 2.0
+// Project: https://github.com/mikemaccana/whois-json
+// Definitions by: Hwai-En Ho <https://github.com/hwaien>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/**
+ * Perform a WHOIS lookup.
+ *
+ * @param domainName - The domain name on which to perform the lookup.
+ * @param options - Options to tweak the behavior of a WHOIS lookup.
+ */
+export default function whois(domainName: string, options?: WhoisOptions): Promise<WhoisResult>;
```

Comment:
This package was declared using `module.exports = ...` or some variant. It should be declared with an `export = ...` like one of the following:

### Variables

```ts
declare var xyz: SomeType;

export = xyz;
```

### Classes

```ts
class Foo {
  // ...
}

export = Foo
```

### Functions

```ts
function foo(): any;

export = foo;
```

To import the entity using a default import, you will need to enable `esModuleInterop`. Alternatively, you can just import using a more appropriate syntax like

```ts
import foo = require("foo");
```

[See more here in the FAQ](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/9f9f6430117d025aef62ed56128deb65508edc86/README.md#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default).

Reply:
Thanks for the pointer!

I rewrote the file using the following as example: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/a2hs.js/index.d.ts